packages/synapse-react-client/src/synapse-queries/entity/useEntityBundle.ts

```diff
@@ -7,17 +11,15 @@ import {
   EntityBundleRequest,
 } from '@sage-bionetworks/synapse-types'
 
-export function useGetEntityBundle<
+export function useGetEntityBundleQueryOptions<
```

Comment:
Extract logic to return query options, which can be passed into either `useQuery` or `useSuspenseQuery`